docs/source/markdown/podman-pod-create.1.md

```diff
@@ -102,19 +102,23 @@ Assign a name to the pod.
 #### **--network**=*mode*, **--net**
 
 Set network mode for the pod. Supported values are
-- **bridge**: Create a network stack on the default bridge. This is the default for rootfull containers.
-- **host**: Do not create a network namespace, all containers in the pod will use the host's network. Note: the host mode gives the container full access to local system services such as D-bus and is therefore considered insecure.
-- Comma-separated list of the names of CNI networks the pod should join.
-- **slirp4netns[:OPTIONS,...]**: use slirp4netns to create a user network stack.  This is the default for rootless containers.  It is possible to specify these additional options:
-  - **allow_host_loopback=true|false**: Allow the slirp4netns to reach the host loopback IP (`10.0.2.2`). Default is false.
+- **bridge**: create a network stack on the default bridge;
+- **none**: no networking;
+- **host**: use the Podman host network stack. Note: the host mode gives the container full access to local system services such as D-bus and is therefore considered insecure;
+- **network**: connect to a user-defined network, multiple networks should be comma-separated;
```

Comment:
remove semi colons?

Reply:
I copied this from podman create. I can remove the semi colons everywhere if you want.